ansible/roles/containerd/README.adoc

```diff
@@ -0,0 +1,48 @@
+== Role: Kubernetes Cluster
+
+Cluster target type: K8s
+
+=== Description
+
+This role will install a Kubernetes cluster on a linux machine using `kubelet` and `kubeadmin`.
```

Comment:
As docker or containerd can be installed part of this role, then the description should mention it instead of the wrong text `This role will install a Kubernetes cluster on a linux machine using `kubelet` and `kubeadmin`.` ;-)

Reply:
improved the description